### DISCUSSION TITLE
新版本出现多余的打印信息和警告

### DISCUSSION THREAD

**Discussion by MangMax** [zouyaoji/vue-cesium#478] [Q&A]
![image](https://github.com/zouyaoji/vue-cesium/assets/68525017/2747d6d6-2119-439b-95bb-89c4d103ee95)
更新到3.2.4版本后会出现一行打印信息，并且会有后面的警告出现，使用最新的cesium版本还是无法消除

**Comment by zouyaoji** [comment]
正常的哈，这是vue-cesium根据最新的cesium版本转发了事件，但你采用的cesium版本可能没这些事件就有警告，只有开发环境会提示。
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/composables/use-events/index.ts#L15

不想提示出来？
